BCH-Wunder-Project-team-4/Wunder.io
# Scroll to top

## Link to ticket:

[top button](https://trello.com/c/BGLGhp1Z/126-go-to-top-button)

## Changes proposed in this PR:

- back to top button
- fixes for issue 58 and 60

## How to test:

1. git fetch origin scrollToTop
2. git checkout scrollToTop
3. Button should be appearing when scrolling down

<img width="322" alt="Screenshot 2023-12-19 at 19 24 11" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/121946942/c15342b7-37bb-4aaa-8aac-92291020266e">
